src/action.js

```diff
@@ -78,6 +78,10 @@ function getHistoryCommits(from, to) {
     options.firstParent = true
   }
 
+  if (getInput('excludeTargetBranch') === 'true') {
+    options[`^${GITHUB_BASE_REF}`] = true
+  }
```

Comment:
Not 100% certain I have the correct variable here since this code has literally never run. (See comment in description about "how to test")